packages/api/schema/stryker-core.json

```diff
@@ -282,18 +282,19 @@
       "$ref": "#/definitions/eventRecorderOptions",
       "default": {}
     },
+    "ignorePatterns": {
+      "description": "Specify the patterns to all files or directories that are not used to run your tests and thus should *not be copied* to the sandbox directory for mutation testing. Each patterns in this array should be a [`.gitignore`-style glob pattern](https://git-scm.com/docs/gitignore#_pattern_format).\n\nThese patterns are **always ignored**: `['node_modules', '.git', '/reports', '/stryker.log', '.stryker-tmp']`. Because Stryker always ignores these, you should rarely have to adjust the `\"ignorePatterns\"` setting at all. If you want to undo one of these ignore patterns, you can use the `!` prefix, for example: `['!node_modules']`.\n\nIf a glob pattern starts with `/`, the pattern is relative to the current working directory. For example, `/foo.js` matches to `foo.js` but not `subdir/foo.js`.\n\nWhen using the command line, the list can only contain a comma separated list of globbing expressions.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "default": []
+    },
     "fileLogLevel": {
       "description": "Set the log level that Stryker uses to write to the \"stryker.log\" file",
       "$ref": "#/definitions/logLevel",
       "default": "off"
     },
-    "files": {
```

Comment:
Shouldn't this still be in the schema, but deprecated?

Reply:
As explained during the community meeting, I don't want people to use it. The rewrite to `ignorePatterns` is being done _before_ the options are validated. This is in line with other deprecated options.